### ISSUE TITLE
not work starlight-image-zoom plugin

### ISSUE THREAD

**Issue by Arhell** [martrapp/astro-vtbot#89]
### Describe the Bug

The plugin doesn't work when I go to the page.
starts working only after page reload

I use
https://events-3bg.pages.dev/jotter/starlight/guide/#step-2-create-the-new-head-component

### What's the expected result?

After going to the page, zoom should work

### Link to Minimal Reproducible Example

Unfortunately locally

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by martrapp** [comment]
Hi @Arhell, I'm sorry that astro-vtbot broke your site. I can not repair this in vtbot's side. But I opened an PR with starlight-image-zoom.

**Comment by Arhell** [comment]
> Hi @Arhell, I'm sorry that astro-vtbot broke your site. I can not repair this in vtbot's side. But I opened an PR with starlight-image-zoom.

Thank you. You've already done a great job. This plugin provides convenience for reading documentation without unnecessary scrolling.

**Comment by HiDeoo** [comment]
Sorry for the inconvenience and thanks a lot for your pull request @martrapp, amazing and efficient work :raised_hands:

I just published [`v0.3.0`](https://github.com/HiDeoo/starlight-image-zoom/releases/tag/v0.3.0) of the `starlight-image-zoom` plugin which includes your changes.

**Comment by Arhell** [comment]
I checked, everything works, thank you very much for your work and efficiency!

**Comment by martrapp** [comment]
Thank you both so much for taking the time for review/approval and test! I really appreciate it!
